src/Core/src/Platform/iOS/MauiButton.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using UIKit;
+
+namespace Microsoft.Maui.Platform
+{
+	internal class MauiButton : UIButton
```

Comment:
Can we make this public? It'll be frustrating for users if they can't reuse the control we're using.

Reply:
Should i change the handler to MauiButton too ?

Reply:
We can't because AFAIK that'd be too breaking of a change. 

pinging @mattleibow just to sanity check.